libcontainer/SPEC.md

```diff
@@ -278,10 +306,18 @@ maximum memory bandwidth of 20% on socket 0 and 70% on socket 1.
 
 "linux": {
     "intelRdt": {
-        "closID": "guaranteed_group",
```

Comment:
why is this removed?

Reply:
This should stay, removed unfortunately during rebase.